hackathon_site/event/static/event/js/script.js

```diff
@@ -0,0 +1,63 @@
+// Change navbar color on scroll
+$(document).scroll(function () {
+    let $nav = $(".navbar");
+    $nav.toggleClass("scrolled", $(this).scrollTop() > $nav.height());
+});
+
+$(document).ready(function () {
+    // Materialize stuff
+    $(".sidenav").sidenav();
+    $(".carousel").carousel({ dist: 0, padding: 600 });
+    setInterval(function () {
+        $(".carousel").carousel("next");
+    }, 3000);
+
+    $(".scrollspy").scrollSpy();
+    $(".collapsible").collapsible();
+
+    // Countdown stuff
+
+    // Registration open and close date, and event date
+    let regOpenDate = new Date("Sep 1, 2020 00:00:00").getTime();
+    let regCloseDate = new Date("Sep 30, 2020 00:00:00").getTime();
+    let eventDate = new Date("Oct 31, 2020 00:00:00").getTime();
+
+    let now = new Date().getTime();
+
+    // Set the title based off what it's counting down to
+    if (regOpenDate > now) {
```

Comment:
You have an edge case when the cutoff dates === now, you may want to make these >=